hr_attendance_management/views/hr_employee.xml

```diff
@@ -41,6 +41,9 @@
                     <field name="extra_hours" widget="float_time"/>
                 </div>
             </field>
+            <xpath expr="//field[@name='work_location']" position="replace">
```

Comment:
Perfect in the form view of hr.employee, but you should as well replace it in the tree view and form view of hr.attendance